app/features/feedback/classifier/helpers/open-feedback-modal.jsx

```diff
@@ -33,10 +33,18 @@ function getFeedbackMarks(feedback) {
     .value();
 }
 
+// The subject viewer will be hidden if `hideSubjectViewer` is enabled on
+// _any_ feedback rule.
+function getHideSubjectViewer(feedback) {
+  return feedback.reduce((result, item) =>
+    (result || item.hideSubjectViewer) || false, false);
```

Comment:
I found this a bit hard to read. Is it logical OR'ing `rule.hideSubjectViewer` for all the rules?